proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -0,0 +1,328 @@
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+}
+
+message GetConnectClusterResponse {
+  ConnectCluster cluster = 1;
+}
+
+message ListConnectClustersRequest {}
+
+message ListConnectClustersResponse {
+  repeated ConnectCluster clusters = 1;
+}
+
+// KafkaConnectService is the service for the Kafka connect, it exposes the
+// Kafka Connect API, you can set multiple Kafka connect services and all of
+// them can be managed using this service definition, the request is not only
+// proxied but also enriched with better error handling and custom
+// documentation and configuration
+service KafkaConnectService {
+  rpc ListConnectClusters(ListConnectClustersRequest) returns (ListConnectClustersResponse) {
+    option (google.api.http) = {get: "/v1alpha1/kafka-connect/clusters"};
+  }
+  // GetConnectClusterInfo implements the get cluster info method, exposes a Kafka
+  // Connect equivalent REST endpoint
+  rpc GetConnectCluster(GetConnectClusterRequest) returns (GetConnectClusterResponse) {
+    option (google.api.http) = {
+      get: "/v1alpha1/kafka-connect/clusters/{cluster_name}"
+      response_body: "cluster"
+    };
+  }
+  // ListConnectors implements the list connectors method, exposes a Kafka
+  // Connect equivalent REST endpoint
+  rpc ListConnectors(ListConnectorsRequest) returns (ListConnectorsResponse) {
+    option (google.api.http) = {get: "/v1alpha1/kafka-connect/clusters/{cluster_name}/connectors"};
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "List connectors"
+      description: "List connectors managed by the kafka-connect service, use the cluster_name defined in the configuration file."
+    };
+  }
+  // CreateConnector implements the create connector method, and exposes an
+  // equivalent REST endpoint as the Kafka connect API endpoint
+  rpc CreateConnector(CreateConnectorRequest) returns (CreateConnectorResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha1/kafka-connect/clusters/{cluster_name}/connectors"
+      body: "connector"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Create connector"
+      description: "Attempt to create a connector with the specific configuration"
+    };
+  }
+  // RestartConnector implements the restart connector method, exposes a Kafka
+  // Connect equivalent REST endpoint
+  rpc RestartConnector(RestartConnectorRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      post: "/v1alpha1/kafka-connect/clusters/{cluster_name}/connectors/{name}/restart"
+      body: "options"
+    };
+  }
+  // GetConnector implements the get connector method, exposes a Kafka
+  // Connect equivalent REST endpoint
+  rpc GetConnector(GetConnectorRequest) returns (GetConnectorResponse) {
+    option (google.api.http) = {
+      get: "/v1alpha1/kafka-connect/clusters/{cluster_name}/connectors/{name}"
```

Comment:
Let's present the URL path in the sync. I can imagine it may be controversial, because:

1. It contains `kafka-connect`. We maybe want to use `managed-connectors` for Cloud, but not for community.
2. We always have a single kafka connect cluster in cloud